code/modules/hydroponics/plant_genes.dm

```diff
@@ -351,12 +351,17 @@
 			to_chat(user, "<span class='notice'>You add some cable to [G] and slide it inside the battery encasing.</span>")
 			var/obj/item/stock_parts/cell/potato/pocell = new /obj/item/stock_parts/cell/potato(user.loc)
 			pocell.icon_state = G.icon_state
-			pocell.maxcharge = G.seed.potency * 20
 
 			// The secret of potato supercells!
 			var/datum/plant_gene/trait/cell_charge/CG = G.seed.get_gene(/datum/plant_gene/trait/cell_charge)
 			if(CG) // 10x charge for deafult cell charge gene - 20 000 with 100 potency.
```

Comment:
Criminally wrong note btw

Reply:
Yeah like I said numbers and how it is done is up to change I dont have enough time to sort it tonight. I didnt know rate was 0.2